src/modules/20-rbac/pages/ResourceGroups/views/ResourceGroupsList.tsx

```diff
@@ -111,6 +111,8 @@ const ResourceGroupsList: React.FC = () => {
           </Layout.Horizontal>
         }
       />
+      {/* eslint-disable-next-line */}
+      {/*@ts-ignore*/}
       <PageBody loading={loading} retryOnError={() => refetch()} error={error ? getRBACErrorMessage(error) : ''}>
```

Comment:
Instead of turning off ESLint, can we just type-assert the `error` as `RBACError`?
```suggestion
      <PageBody loading={loading} retryOnError={() => refetch()} error={error ? getRBACErrorMessage(error as RBACError) : ''}>
```

Reply:
I see you did the other place that does this. Can we do it here too?

Reply:
+1

Reply:
So when i rebased the same thing was being done here now as part of someone elses changes

Reply:
Have fixed this now